lma-browser-extension-stack/public/content_scripts/providers/teams.js

```diff
@@ -0,0 +1,271 @@
+console.log("Inside LMA Teams script");
+
+let metadata = {
+  baseUrl: window.location.origin
+}
+
+const openChatPanel = function () {
+  const chatPanelButton = document.getElementById('chat-button');
+  // click chatPanelButton if found
+  if (chatPanelButton) {
+    chatPanelButton.click();
+  }
+}
+
+const activeRingClass = "fui-Avatar r81b29z ___l339ri0";
+const inactiveRingClass = "fui-Avatar r81b29z ___1okzwt8";
+
+const sendChatMessage = function (message) {
```

Comment:
This piece doesn't work yet, right?  Let's add a comment in the code to that effect..and a TODO comment to come back to it..